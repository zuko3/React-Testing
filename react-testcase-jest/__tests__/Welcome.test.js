import React from "react";
import { mount } from "enzyme";
import Welcome from "../Welcome";

describe("cheeck the Welcome", () => {
  it("should check the welcome structure", () => {
    const wrapper = mount(<Welcome />);
    expect(wrapper).toMatchSnapshot();
  });
  it("Welcome renders hello world", () => {
    const welcome = mount(<Welcome />);
    expect(welcome.find("div").text()).toEqual("Hello world");
  });
});
