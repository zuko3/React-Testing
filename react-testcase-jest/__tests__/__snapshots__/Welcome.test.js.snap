// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cheeck the Welcome should check the welcome structure 1`] = `ReactWrapper {}`;
